src/eventSubscription/registerEventSubscriptionCommands.ts

```diff
@@ -17,9 +21,14 @@ export function registerEventSubscriptionCommands(): void {
     ext.context.subscriptions.push(ext.eventSubscriptionTree);
     ext.context.subscriptions.push(vscode.window.registerTreeDataProvider('azureEventGridSubscriptionExplorer', ext.eventSubscriptionTree));
 
+    ext.context.subscriptions.push(vscode.languages.registerCodeLensProvider({ language: 'json', pattern: '**/*.[eE][vV][eE][nN][tT][gG][eE][nN][eE][rR][aA][tT][oO][rR].[jJ][sS][oO][nN]' }, new MockCodeLensProvider()));
```

Comment:
Should you support jsonc?

Reply:
Done. I think the glob pattern is super ugly, but it works for me and I couldn't find anything cleaner. Let me know if you know a better way